src/config/hintr_version

```diff
@@ -1 +1 @@
-master
+mrc-4673
```

Comment:
Should we merge this hintr PR?

Reply:
i think we should merge it just after we flip the feature switch, to make sure there arent any problems, i dont see why there would be but just to be on the safe side